recipes/newick_utils/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "1.6" %}
+{% set sha256 = "2c142a2806f6e1598585c8be5afba6d448c572ad21c142e70d6fd61877fee798"  %}
+
+
+
+package:
+  name: newick_utils
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  fn: newick-utils-{{ version }}.tar.gz
+  url: http://cegg.unige.ch/pub/newick-utils-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+  - gcc  # [not osx]
+  - llvm # [osx]
+  - libxml2 2.9.*
+   
+  run:
+  - libgcc # [not osx]
+  - icu 58.*
+  - libiconv
+  - backports.lzma
```

Comment:
Sorry it's just included when requiring libxml2 I have cleaned the dependencies.